.github/actions/setup-kind/action.yml

```diff
@@ -0,0 +1,16 @@
+name: Setup Kind
```

Comment:
Is it necessary to install `kind`? It's always pre-installed on GitHub CI runner.

Reply:
Cool